paddle/fluid/platform/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ ENDIF()
 cc_library(cpu_info SRCS cpu_info.cc DEPS ${CPU_INFO_DEPS})
 cc_test(cpu_info_test SRCS cpu_info_test.cc DEPS cpu_info)
 
-nv_library(gpu_info SRCS gpu_info.cc DEPS gflags glog enforce monitor)
+nv_library(gpu_info SRCS gpu_info.cc DEPS gflags glog enforce monitor dynload_cuda)
```

Comment:
any reason behind this change?

Reply:
this is because we use Hascudnn in gpu_info

Reply:
so an existing bug unrelated to this PR?